tox.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:20.04
+
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y apt-transport-https
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y software-properties-common && add-apt-repository ppa:deadsnakes/ppa
+
+# ensure the base image has what we need
+#RUN DEBIAN_FRONTEND=noninteractive apt-get -y install build-essential software-properties-common
```

Comment:
Do you need this anymore or want to keep for reference?

Reply:
Hmm, no, don't need it any more as far as I can tell. I'll remove it so I don't leave commented out stuff!